query.proto

```diff
@@ -7,6 +7,21 @@ option go_package = "zvelo.io/msg";
 import "zvelo/msg/dataset.proto";
 import "zvelo/msg/status.proto";
 
+message QueryResultRequest {
```

Comment:
`QueryResultRequest`??  Wouldn't it be better to make `request_id` into a GRPC message for the way in, and then reuse that for polling?